File: day1-first-app/day1-first-app/ViewController.m
//
//  ViewController.m
//  day1-first-app
//
//  Created by Dave Henson on 6/24/15.
//  Copyright (c) 2015 Dave Henson. All rights reserved.
//

#import "ViewController.h"

@implementation ViewController

-(void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
{
    redView.center = CGPointMake(300, 300);
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    redView = [[UIView alloc] init];
    redView.frame = CGRectMake(20, 20, 100, 100);
    //redView.backgroundColor = [UIColor redColor];
    redView.backgroundColor = [UIColor colorWithRed:1.0 green:0.0 blue:0.0 alpha:0.5];
    [self.view addSubview:redView];

    UIView *blueView = [[UIView alloc] init];
    blueView.frame = CGRectMake(10, 10, 30, 30);
    blueView.backgroundColor = [UIColor blueColor];
    [redView addSubview:blueView];
    
    //s = [[Student alloc] init];
    s = [Student createANewStudent];
    
    [s sendAnEmail:@"hello"
              body:@"this is the body"
          priority:1
              from:@"dh@msn.com"
         recipient:@"joe@nowhere.com"];
    
}

@end

File: day1-swift-demo/day1-swift-demo/ViewController.swift
//
//  ViewController.swift
//  day1-swift-demo
//
//  Created by Dave Henson on 6/24/15.
//  Copyright (c) 2015 Dave Henson. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var myView:UIView = UIView()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        myView = UIView(frame: CGRectMake(10.0, 10.0, 100.0, 100.0))
        self.view.addSubview(myView)
        myView.backgroundColor = UIColor(red: 1.0, green: 0.0, blue: 0.0, alpha: 1.0)
        
        
    }

    override func touchesBegan(touches: Set<NSObject>, withEvent event: UIEvent) {
        
        for touch : AnyObject in touches
        {
            let location = touch.locationInView(self.view)
            myView.center = location
        }
    }


}

